spec/text_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'text'
+
+describe Text do
+
+  subject(:text) { described_class.new }
+
+  describe '#send_message' do
+
+    it 'sends a text message' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.